trinity/plugins/registry.py

```diff
@@ -30,10 +30,15 @@ def is_ipython_available() -> bool:
 # we'll be able to load plugins from some path and control via Trinity
 # config file which plugin is enabled or not
 
-ENABLED_PLUGINS = [
+BUILTIN_PLUGINS = [
     AttachPlugin() if is_ipython_available() else AttachPlugin(use_ipython=False),
     FixUncleanShutdownPlugin(),
     JsonRpcServerPlugin(),
     LightPeerChainBridgePlugin(),
     TxPlugin(),
 ]
+
+# To enable discovery plugins need to define entrypoints at 'trinity.plugins'
+# https://packaging.python.org/guides/creating-and-discovering-plugins/#using-package-metadata
+DISCOVERED_PLUGINS = [
+    entry_point.load() for entry_point in pkg_resources.iter_entry_points('trinity.plugins')]
```

Comment:
Maybe let's put a `ALL_PLUGINS = BUILTIN_PLUGINS + DISCOVERED_PLUGINS` here and then just import `ALL_PLUGINS` in `main.py`

Reply:
Done